packages/commonwealth/client/scripts/controllers/server/types.ts

```diff
@@ -0,0 +1,48 @@
+// Types for handling server API requests from the client side.
+
+import { Request, Response } from 'express';
```

Comment:
Why are we importing express types on the client?

Reply:
removed. We supposed to assume we have visual of the models, which I did not understand at first